packages/teleport/src/Databases/fixtures/index.ts

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { Database } from 'teleport/services/database';
+
+export const databases: Database[] = [
+  {
+    name: 'aurora',
+    desc: 'PostgreSQL 11.6: AWS Aurora ',
+    protocol: 'postgres',
+    type: 'rds',
+    uri:
+      'postgres-aurora-instance-1.c1xpjrob56xs.us-west-1.rds.amazonaws.com:5432',
+    tags: ['cluster: root', 'env: aws'],
```

Comment:
actually, could we keep this as `labels`? I was mulling it over and not sure how `tags` originated since label is more accurate

Reply:
lets get @alex-kovoy approval before you start renaming.. only to rename it back ;)

Reply:
I agree, the [teleport docs refer to them as labels ](https://goteleport.com/docs/admin-guide/#labeling-nodes-and-applications) so it's more accurate. In #268 `tags` was being used so I just applied the same. I will also change the `NodeList` logic to use `labels` instead of tags so things are more consistent :slightly_smiling_face:

Reply:
Sounds good, I just reverted that commit